### ISSUE TITLE
is this project being maintained?

### ISSUE THREAD

**Issue by ppazos** [syslog4j/syslog4j#2]

**Comment by ssill2** [comment]
bump?

**Comment by ppazos** [comment]
I don't think it is maintained and the CI build is failing, might need to look for other implementations.

**Comment by ssill2** [comment]
yeah,  I think I'm going to go a different route.

Since the machines my apps that use this on run rsyslog and only certain  app messages need to be sent to syslog I might use the imhttp input module in ryslog to allow my app to send an http message to rsyslog instead of connect via syslog.  It doesn't seem worth writing my own JNI/JNA wrapper around syslog.  I just used this lib since it was already there and did what I needed.

**Comment by sean-scott-lr** [comment]
have you guys found an alternate to this project?

**Comment by sean-scott-lr** [comment]
Also, I wonder if this was apache or MIT licensed if it would see more community support?

**Comment by ssill2** [comment]
I came up with my own alternative.  I wrote a jna access layer to the syslog methods in libc.

**Comment by ssill2** [comment]
this logs to local rsyslog via domain socket and you can then route them anywhere else you want them to go via rsyslog configuration.

**Comment by ssill2** [comment]
after I did it I found
https://github.com/m-creations/log4j2-native-syslog/blob/master/src/main/java/com/mcreations/log4j/Clib.java

Which is basically what I did.  I just didn't go to the level of making a generic logger/appender for it.

**Comment by sean-scott-lr** [comment]
Ah, thanks. I am looking for a solution for the syslog Server side.

Thanks!
